R/addLDA.R

```diff
@@ -0,0 +1,105 @@
+#' Latent Dirichlet Allocation
+#'
+#' These functions perform Latent Dirichlet Allocation on data stored in a 
+#'  \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'  object.
+#' 
+#' @param x a \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'  object.
+#'   
+#' @param k Integer, number of latent vectors/topics 
+#'   (By default: \code{2}).
```

Comment:
We are trying to harmonize these descriptions little bit. This could be:

Integer scalar. A number of latent vectors/topics. (Default: \code{2})

The type. Detailed description. Default value.